ceph_iscsi_config/lun.py

```diff
@@ -574,6 +581,38 @@ def activate(self):
             if client_err:
                 raise CephiSCSIError(client_err)
 
+    def _erasure_pool_check(self):
+        if not self.datapool:
+            return None
+
+        data, err = run_shell_cmd(
+            "ceph -n {client_name} --conf {cephconf} osd metadata --format=json".
+            format(client_name=settings.config.cluster_client_name,
+                   cephconf=settings.config.cephconf))
+        if err:
+            self.logger.error("Cannot get the objectstore type")
+            return err
+        bluestore = False
+        for _osd in json.loads(data):
+            store_type = _osd['osd_objectstore']
+            self.logger.debug(f"objectstore type is ({store_type})")
+            if store_type == 'bluestore':
+                bluestore = True
+                break
+
+        if not bluestore:
+            return None
```

Comment:
Did you intend for filestore (or any other objectstore except bluestore) to not produce an error?  This entire function seems buggy to me because when it is called, `None` is interpreted as success.

Reply:
Will fix it.